.github/workflows/build-mac.yml

```diff
@@ -0,0 +1,57 @@
+name: Mac package
+
+on: [push, pull_request]
+
+jobs:
+  package:
+    runs-on: macos-10.15
+    name: Build and package for Mac OS
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install and package
+        run: |
+          /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" # install Homebrew
+          brew install enchant # This installs Python 3.9 for some reason, which we don't want
+          brew uninstall --ignore-dependencies python@3.9
+          brew install git pygobject3 gtk+3 python@3.8 gtksourceview3 adwaita-icon-theme sdl cmake
+          PATH=/usr/local/opt/python@3.8/bin:/usr/local/bin:$PATH
+
+          # Install armips
+          git clone --recursive https://github.com/Kingcom/armips.git
+          mkdir armips/build && cd armips/build
+          cmake -DCMAKE_BUILD_TYPE=Release ..
+          cmake --build .
+          cd - # Go back to the SkyTemple directory
+          cp armips/build/armips ./installer/armips
+
+          # Install skytemple-eventserver to have a compatible version that satifies the requirements.txt
+          git clone https://github.com/SkyTemple/skytemple-eventserver.git
+          pip3 install -r ./skytemple-eventserver/requirements.txt
+          pip3 install ./skytemple-eventserver
+
+          # Install other dependencies and SkyTemple itself
+          pip3 install py-desmume skytemple-rust pyinstaller
+          pip3 install -r requirements-mac.txt
+          pip3 install .
+
+          # Install the McMojave theme
+          cd installer
+          curl https://cdn.discordapp.com/attachments/712341699292037121/784474569884303380/McMojave.zip -o McMojave.zip
+          unzip McMojave.zip > /dev/null
+
+          pip3 install pyinstaller
+
+          # Package
+          current_version=${GITHUB_REF##*/} # https://github.community/t/how-to-get-just-the-tag-name/16241
```

Comment:
Not sure if this actually works on a release, it's supposed to extract the version from the tag (the VERSION file contains `master` on non-release builds by the way).